build/Dockerfile

```diff
@@ -34,27 +46,17 @@ RUN mkdir -p $DST_HOME \
 # Copy common scripts.
 COPY /script/* /usr/local/bin/
 
-# Copy entrypoint script.
-COPY /docker-entrypoint.sh /entrypoint.sh
-
 # Copy static files.
 COPY /static $DSTA_HOME
+
+# Create pipe for the game console.
 RUN mkfifo $DSTA_HOME/console \
 	&& chown -R $STEAM_USER:$STEAM_USER $DSTA_HOME \
-	&& mkdir -p `dirname $CLUSTER_PATH` \
-	&& chown $STEAM_USER:$STEAM_USER `dirname $CLUSTER_PATH`
+	&& mkdir -p $(dirname $CLUSTER_PATH) \
+	&& chown $STEAM_USER:$STEAM_USER $(dirname $CLUSTER_PATH)
 
-# Set environment variables.
-ENV DESCRIPTION="Powered by DST Academy." \
-	SERVER_PORT="10999" \
-	MAX_PLAYERS=4 \
-	GAME_MODE=survival \
-	VOTE_KICK_ENABLE=true \
-	PAUSE_WHEN_EMPTY=true \
-	SHARD_NAME=shard \
-	SHARD_IS_MASTER=false \
-	SHARD_BIND_IP="0.0.0.0" \
-	BACKUP_LOG_COUNT=0
+# Copy entrypoint script.
+COPY /docker-entrypoint.sh /
```

Comment:
Moved this down here and I thought *not`renaming this file is better. Some other images do it the same way. Seems cleaner to me. Oppinions?